.github/workflows/yaml-checks.yaml

```
@@ -0,0 +1,26 @@
+name: YAML checks
+
+on:
+  push:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - '**.yaml'
+      - '**.yml'
+  pull_request:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - '**.yaml'
+      - '**.yml'  
```

should we also add here `.yamllint`?
```suggestion
      - '**.yaml'
      - '**.yml'  
      - '**.yamllint'  
```

---

Yes! We can add `.yamllint` here. I'll do this.